BitwardenShared/Core/Platform/Services/StateService.swift

```diff
@@ -1159,6 +1159,15 @@ enum StateServiceError: Error {
 
     /// The user has no user key.
     case noEncUserKey
+
+    var errorDescription: String? {
```

Comment:
🔨 missed docs here

Reply:
This is an implementation of the `LocalizedError` protocol method. Usually we don't document protocol implementations; think it's needed here?

Reply:
oh yeah i see now, my mistake!